### DISCUSSION TITLE
I cant seem to render text.

### DISCUSSION THREAD

**Discussion by ElectricIceBird** [gurkenlabs/litiengine#788] [Q&A] [answered]
package pkg2d.platformer.game;
import de.gurkenlabs.litiengine.Game;
import java.awt.Color;
import java.awt.Graphics2D;
import java.awt.geom.Rectangle2D;
import java.awt.image.BufferedImage;

/**
 *
 * @author KTECH
 */
public class Main {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        // TODO code application logic here

  Game.init(args);
Graphics2D g = null;
Game.graphics().renderText(g, "my text", 50, 100);

        Game.loop();

    }
    
}
Please assist

**Comment by nightm4re94** [comment]
You are passing null as the Graphics2D object to draw on.
The proper way would be to create and add a Screen, then in that screen's render method take the Graphics2D instance to draw whatever you want.

**Reply by ElectricIceBird** [reply]
Any small amount of code

**Reply by ElectricIceBird** [reply]
how can i do that? The documentation is a bit vague

**Reply by nightm4re94** [reply, marked as answer]
https://litiengine.com/docs/tutorials/creating-a-platformer/

**Reply by ElectricIceBird** [reply]
Thank you for your reply it worked!!

**Reply by nightm4re94** [reply]
Awesome! 👍
